src/components/CustomErrorComponent/CustomErrorComponent.tsx

```diff
@@ -0,0 +1,21 @@
+import fallingMan from '/falling-man.png';
+import { ErrorResponse } from '@bsv/spv-wallet-js-client';
+
+export interface CustomErrorComponentProps {
+  error: ErrorResponse;
+}
+
+export const CustomErrorComponent = ({ error }: CustomErrorComponentProps) => {
```

Comment:
We was talking about some different approach, 
instead of big error page, it should display the empty table and a Toast with error.

For now let's go with this approach to finish MVP faster.

Reply:
It was an additional discussion in slack about that.